package com.training.warehouse.exception.handler;

public class ExceptionMessage {
    public static final String UNKNOWN = "something went wrong";
    public static final String USER_NOT_FOUND = "user not found";
    public static final String UNAUTHORIZED = "unauthorized";
    public static final String FOBBIDEN = "forbidden";
    public static final String USER_ALREADY_EXIST = "user already exist";
    public static final String EMAIL_ALREADY_EXIST = "email already exist";
    public static final String FILENAME_IS_NOT_VALID = "file name is not valid";
    public static final String FILE_PATH_IS_NOT_VALID = "file path is not valid";
    public static final String FILETYPE_NOT_ALLOWED = "file type is not allowed";
    public static final String INBOUND_NOT_FOUND = "inbound not found";
    public static final String CANNOT_DELETE_INBOUND = "cannot delete inbound because it has outbound";
    public static final String OUTBOUND_NOT_FOUND = "outbound not found";
    public static final String OUTBOUND_CONFIRMED = "outbound was confirmed";
}
